ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type.schema.yml

```
@@ -868,6 +868,48 @@ $defs:
               This setting is applicable to L2 switches (switches using port-channel trunks as uplinks).
             type: str
             format: ipv4
+          inband_mgmt_ipv6_address:
+            documentation_options:
+              table: node-type-inband-management-configuration
+            description: |
+              IPv6 address assigned to the inband management interface set with 'inband_mgmt_vlan'.
+              This overrides 'inband_mgmt_ipv6_subnet', hence all behavior of 'inband_mgmt_ipv6_subnet' is removed.
```

sounds like we are saying the `ubnet` key iis being `removed` which means something in our schema.
```suggestion
              This overrides 'inband_mgmt_ipv6_subnet', hence the configuration of 'inband_mgmt_ipv6_subnet' is ignored.
```

---

Actually it is still used for the gateway reading through the code, so probably need t o reword